compass/hierarchy.py

```diff
@@ -145,12 +43,10 @@ def get_hierarchy(self, compass_id: int, level: str) -> dict:
         """Recursively get all children from given unit ID and level, with caching"""
```

Comment:
Outside the scope of this PR - but would be good to think a little around the caching implementation. Currently this 'cache' never expires unless it is removed. Lets get the data stuff right then can think about caching/rate limiting etc.

Reply:
Could you add this as an issue? Great point but easy to loose in this PR!

Reply:
please*